src/main/resources/roleconfig/role_common.json

```diff
@@ -2126,14 +2155,47 @@
         },
         "grantType": {
           "mandatory": true,
-          "values": ["BASIC"]
+          "values": ["STANDARD"]
+        },
+        "classification": {
+          "mandatory": true,
+          "values": ["PUBLIC"]
+        },
+        "attributes": {
+          "jurisdiction": {
+            "mandatory": true,
+            "values": ["PUBLICLAW"]
+          }
+        }
+      }
+    ]
+  },
+  {
+    "name": "hearing-viewer",
```

Comment:
hearing-viewer for CTSC appearing two times. one for publiclaw and another for privatelaw. can you please merge as one